aws/kops-iam-users/data.tf

```diff
@@ -0,0 +1,37 @@
+variable "kops_admins_data" {
+  type        = "list"
+  default     = []
```

Comment:
I want to deprecate this pattern - so let's not inherit it here.

Reply:
@osterman removed and will be explicit in https://github.com/cloudposse/root.cloudposse.co/blob/master/conf/users/osterman.tf#L5